### ISSUE TITLE
Drop all FAQs in the repository and merge them into the wiki FAQ page

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3539]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)| @thijstriemstra reported|
|-|-|
|Trac ID|trac#3539|
|Type|task|
|Created|2008-11-16 14:43:57Z|
|Branch|https://github.com/twisted/twisted/tree/drop-faqs-3539|

We've stopped maintaining FAQs in the repository.  Instead, we maintain one on the wiki - FrequentlyAskedQuestions.  Some subprojects still have FAQs, though.  We should remove them from the repository and put them onto the wiki.

<details><summary>Searchable metadata</summary>

```
trac-id__3539 3539
type__task task
reporter__thijs thijs
priority__normal normal
milestone__ 
branch__branches_drop_faqs_3539 branches/drop-faqs-3539
branch_author__thijs thijs
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1226846637000000 1226846637000000
changetime__1238544090000000 1238544090000000
version__None None
owner__ 
cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_1"></a>|@thijstriemstra set owner to @thijstriemstra<br>@thijstriemstra set status to `assigned`|
|-|-|

Renamed the ticket description based on the IRC chat, pasted below:

```
[14:46] exarkun: thijs: How about we don't distribute the FAQ?
[14:47] thijs: what about the t.web faq.. then we should get rid of that one as well, merge it into the wiki page?
[14:48] exarkun: probably
[14:48] thijs: exarkun: sounds good not to distribute it, but lets not make an exception for any other faqs in doc/ ?
[14:48] thijs: ok
[14:48] exarkun: I would just throw away the woven faq entries, though.
[14:49] thijs: ill rename the ticket to drop any other faqs and merge them into the wiki, except for woven
```

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun commented|
|-|-|

The old description was:

```
Now that the core FAQ isn't maintained in the Subversion repository anymore (it was dropped with #3236), we could export the wiki's FrequentlyAskedQuestions page and include it in new releases.
```

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_3"></a>|@thijstriemstra commented|
|-|-|

(In [26505]) Branching to 'drop-faqs-3539'

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_4"></a>|@thijstriemstra commented|
|-|-|

Some entries I'm moving from [doc/web/howto/faq.xhtml](doc/web/howto/faq.xhtml) to FrequentlyAskedQuestions:

----

### Can Twisted Web do PHP?
Yes. It works out-of-the-box, so long as you've got the standalone PHP interpreter installed.

### And can Twisted Web do virtual hosting?
You can decide to go with one big process for all of them, a front server and a seperate server for each virtual host (for example, for permission reasons), and you can even mix-and-match between Apache and Twisted (for example, put Apache in the front and have Twisted handle some subset of the virtual host).

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_5"></a>|@thijstriemstra commented|
|-|-|

(In [26506]) Drop web faq, refs [#3539](https://github.com/twisted/twisted/issues/3539)

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_6"></a>|@thijstriemstra removed owner<br>@thijstriemstra set status to `new`|
|-|-|

Putting the branch and items from comment 4 in this ticket up for review.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_7"></a>|@thijstriemstra commented|
|-|-|

[doc builder](http://buildbot.twistedmatrix.com/builders/documentation/builds/360) is happy as well.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_8"></a>|@exarkun set owner to @thijstriemstra|
|-|-|

Where'd the web faq entries go?  Did they just get deleted?  I looked for them on the FrequentlyAskedQuestions page but did not see them.  And I don't see a dedicated Web FAQ page.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_9"></a>|@thijstriemstra set owner to @exarkun|
|-|-|

Replying to [exarkun](#note_8):
> Where'd the web faq entries go?  Did they just get deleted?  I looked for them on the FrequentlyAskedQuestions page but did not see them.  And I don't see a dedicated Web FAQ page.

I only found 2 entries from that FAQ that seemed helpful. They're listed in [comment:4 comment 4 of this ticket]. Since there are only 2 entries I don't think it's necessary to create a dedicated FAQ page for web. Putting it up for review, let me know your thoughts, cheers.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_10"></a>|@exarkun set owner to @thijstriemstra|
|-|-|

I think it's fine to create a FAQ with just two entries on it.  It'll grow more entries eventually. :)  So if you agree, go ahead and create a FAQ for those and merge the branch.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_11"></a>|@thijstriemstra set status to `assigned`|
|-|-|

Alright, creating TwistedWeb/FrequentlyAskedQuestions

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_12"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [26589]) Merge drop-faqs-3539: drop the last faq in the repository and merge it into the TwistedWeb/FrequentlyAskedQuestions wiki page.

Author: thijs
Reviewer: exarkun
Fixes: [#3539](https://github.com/twisted/twisted/issues/3539)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
